manifests/e/e/Chess/1.0.0.0/e.Chess.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: e.Chess
+PackageVersion: 1.0.0.0

```

```suggestion
PackageVersion: 1.8
```
When updating this value, be sure to change the path of the manifest to `./../1.8/<file>.yaml`